src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerbefund/ProAnalytQuantitativesErgebnisElementConverter.java

```diff
@@ -0,0 +1,26 @@
+package org.ehrbase.fhirbridge.ehr.converter.specific.virologischerbefund;
+
+import org.ehrbase.fhirbridge.ehr.opt.virologischerbefundcomposition.definition.ProAnalytQuantitativesErgebnisChoice;
+import org.ehrbase.fhirbridge.ehr.opt.virologischerbefundcomposition.definition.ProAnalytQuantitativesErgebnisElement;
+import org.hl7.fhir.r4.model.Observation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProAnalytQuantitativesErgebnisElementConverter {
+
+    public ProAnalytQuantitativesErgebnisElement convert(Observation observation){
+
+        ProAnalytQuantitativesErgebnisElement proAnalytQuantitativesErgebnisElement = new ProAnalytQuantitativesErgebnisElement();
+        List <ProAnalytQuantitativesErgebnisChoice> proAnalytQuantitativesErgebnisChoiceList = new ArrayList<>();
+        if (observation.getValueQuantity().hasUnit()){
```

Comment:
Unit is 1..1 no need for check if i see right.
So it will be always DvQuantity. 
Leave it be but maybe add an comment so people will now this is just an double check.

Reply:
Comment added.
https://github.com/ehrbase/fhir-bridge/pull/392/files#diff-3c21d1009f645927c78845f16c33a6c053c9c18394502f2ee4a12c28c9f28e80